core/domain/exp_services_test.py

```diff
@@ -1024,15 +1038,21 @@ def setUp(self):
             }],
             'outcome': {
                 'dest': self.init_state_name,
-                'feedback': ['Try again'],
+                'feedback': {
+                    'audio_translations': {},
+                    'html': 'Try again'
+                },
                 'param_changes': []
             },
             'correct': False,
         }]
         # Default outcome specification for an interaction.
         self.interaction_default_outcome = {
             'dest': self.init_state_name,
-            'feedback': ['Incorrect', '<b>Wrong answer</b>'],
+            'feedback': {
+                'audio_translations': {},
+                'html': 'Incorrect'
```

Comment:
A small, completely optional nit, but consider keeping the `<b>` and `</b>` as a reminder to the reader that this is HTML and not plaintext.

Reply:
done